graphql/util.go

```diff
@@ -770,7 +770,7 @@ func canRestartTask(ctx context.Context, at *restModel.APITask) (*bool, error) {
 	if err != nil {
 		return nil, err
 	}
-	nonrestartableStatuses := []string{evergreen.TaskStarted, evergreen.TaskUnstarted, evergreen.TaskUndispatched, evergreen.TaskDispatched, evergreen.TaskInactive}
+	nonrestartableStatuses := evergreen.TaskUncompletedStatuses
```

Comment:
It seems like this doesn't need a separate variable if it's only used on the next line.

Reply:
+1